scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -357,16 +358,32 @@ function doAddPlugin() {
     fi
   fi
 }
-function doStartEclipse() {
+
+function doLaunchEclipse() {
   doConfigureEclipse -u
   echo "launching Eclipse in workspace ${WORKSPACE} at ${WORKSPACE_PATH}"
   if doIsWindows
   then
     # shellcheck disable=SC2086
-    start "eclipse" /B "${ECLIPSE}" -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${ECLIPSE_JAVA_HOME}/bin/javaw" -showlocation "${WORKSPACE}" -vmargs ${ECLIPSE_VMARGS}
+    start "eclipse" /B doStartEclipse -clean -data "${WORKSPACE_PATH}" -showlocation "${WORKSPACE}"
```

Comment:
This is not working! When we use `doStartEclipse`, the application starts without the custom plugins... the wrapper of eclipse with parameters like `keyring`, `configuration`, `vm` and `vmargs` can not be used easily, because in all places where it is used, further parameters have to be added... does it make sense to still add this wrapper?